Patch 1

titbtcqash opened the pull request:
I changed all of the files in /numerai-tournament/
I have added an appropriate Japanese translation reflecting the contents of the latest English version of the DOCS.

On the other hand, there seems to be no Japanese DOCS for Feature Neutral Correlation and Numerai tournament FAQ. Can I make them?

didy-kpn commented on SUMMARY.md:
```diff
@@ -1,25 +1,25 @@
+* [評価方法について](numerai-tournament/reputation.md)
+* [新規参加者向けのTips](numerai-tournament/new-users.md)
+* [報酬プログラム](numerai-tournament/bounties.md)
+* [フォーラム](numerai-tournament/forum.md)
```
@ageonsen @titbtcqash 

It's very small things, but `connect-index` is still on alphabet. I think it would be better to standardize on katakana or alphabet.

とても細かいですが、 `connect-index.md` は アルファベットのままです。カタカナかアルファベットのどちらかに統一してもいいと思いました。

https://github.com/numerai/docs-jp/pull/3/files#diff-6b0f0247b47401a521caaf3b4eb4646197ee5862095a7cfa1627dfc94b721f7dR5

titbtcqash replied:
ありがとうございます！

didy-kpn commented on SUMMARY.md:
```diff
@@ -1,25 +1,25 @@
+* [Numerai Signalsについての概要](numerai-signals/signals-overview.md)
+* [新規参加者向けのTips](numerai-signals/new-users.md)
 
 ## Connect
```
@ageonsen @titbtcqash 

I think it would be better to Japanese.

ここは日本語で「連絡」でもいいと思いました。

titbtcqash replied:
見出しを修正しましたが、反映されてないですかね？　地味にSignalsをシグナルスと訳すか、信号と訳すか、どうするか迷ってます・・。

ageonsen replied:
シグナルスが良いと思います。Signals の方が良い感じがしたのでそのままにしていました。
信号という表現は聞かず、違和感を感じます。

それはそうと、直訳の修正ありがとうございます。

didy-kpn commented on numerai-tournament/new-users.md:
```diff
@@ -1,43 +1,89 @@
+)<br>
+
+【Twitter】<br>
+[Numeraiに関する日本語での初ツイート](https://twitter.com/khyh_/status/683196065416220672)<br>
```
@ageonsen  @titbtcqash 

公式ドキュメントに日本語での初ツイートへのリンクは不要だと思います。

titbtcqash replied:
あざっす。ついでに2chの奴も消しておきました。　以前書いたQiita記事をそのまま載せたので、公式にふさわしいもののみを載せた方が良いでしょうね

didy-kpn commented on connect/connect-index.md:
```diff
@@ -1,10 +1,11 @@
-* [Twitter](https://twitter.com/numerai) Numerai の公式Twitter アカウント
-* [Medium](https://medium.com/numerai) Numerai blog
-* [r/numerai](https://www.reddit.com/r/numerai/) Numerai の公式Subreddit
+# 連絡先一覧
```
@ageonsen @titbtcqash 

`SUMMARY.md` では 「Coneect Index」が「Numerai運営への連絡」になっていますね。 これどちらがいいんでしょうか...

titbtcqash replied:
connect-index.mdとほかのmdファイルのタイトル整合性をとりました。ありがとうございます！

didy-kpn commented on numerai-tournament/tools-and-libraries.md:
```diff
@@ -1,38 +1,25 @@
+[その他のツール](https://docs.numer.ai/tournament/tools)<br>
+英語版のツールに関するリンク集です。
+
+#### APIクライアント(＊英語版\)
```
@titbtcqash @ageonsen 

下のAPIクライアントの名前の横にだけ:(コロン)があります。こちらは不要でしょうか？

titbtcqash replied:
消し忘れです！修正しました

didy-kpn commented on numerai-tournament/staking-and-payouts.md:
```diff
@@ -1,71 +1,69 @@
+ペイアウトは以下の式で計算されます。<br>
+payout = stake_value * payout_factor * (corr * corr_multiplier + mmc * mmc_multiplier)<br>
+stake_value:ラウンド開始時点の最初の木曜日にステークしたNMRの量<br>
+payout_factor:30万NMR以下では1、30万NMR以上では以下の図に示す値をとります。Numeraiはペイアウトの上限を決めることで持続的なトーナメントの開催を行うことができます。<br>
```
@titbtcqash @ageonsen 

こちらは私だけかもしれないので、不要ならこのままで大丈夫です....

> 30万NMR以下では1、30万NMR以上では以下の図に示す値をとります。

上の文を下記の通りに二つの文にした方が読みやすいと思いました。

「30万NMR以下では1になります。30万NMR以上では以下の図に示す値をとります」

titbtcqash replied:
反映しました！

titbtcqash commented:
I finished proofreading the page of Numerai signals. 
Translated some parts that did not make sense in Japanese, added the description of the latest English version of DOCS, and replaced the images.
I have finished proofreading the content that exists in JPDocs, and I would like you to review it.

ageonsen commented:
Thank you for proofreading!

> 最初の提出が遅れるか、失敗した場合、`example_predictions`と同等のスコアが与えられます。また、予測ファイルが提出されなかった場合、`-0.1`という非常に低いスコアが与えられます。

この部分は、原文が

> The first late or missed submission will receive the score equivalent to the example_predictions. Subsequent late or missed submissions will receive a very low score of -0.1.

なので、

> 最初の提出が遅れるか、失敗した場合、`example_predictions`と同等のスコアが与えられます。それ以降、予測ファイルが提出されなかった場合、`-0.1`という非常に低いスコアが与えられます。

の方が正しいと思うのですがいかがでしょうか。

titbtcqash commented:
修正しました。仰せの通りです。
ありがとうございます！

余談ですが、英語版DOCSは初見者には情報量が少ない部分があったので、追記・意訳を大量に追加しています。
例えば、バーンに関する説明（tournament-overview.md）では、
スコアが高いほど、より多くのNMRを得ることができます。もし負のCorr,MMCとなった場合、ステークしたNMRの一部が没収され、バーンされます。
バーンとはERC-20トークンの持つ機能の一つであり、トークンを永遠に使用できなくする操作のことです。

などです。

原文への忠実さよりも、初見の人が理解できることに重きを置いた方が良い、という判断です。（利用規約等でそれをやったら問題ですが）

ageonsen commented:
コメント反映ありがとうございます！

ERC-20トークンについてはまだまだ勉強中なので、コメントに追記できないままでいました。
追記・意訳大変ありがとうございます。

今回の変更で、初見の人にもわかりやすいものになったと思います。

ちなみに、私にはmergeする権限がないようなので、ある程度完成したらNJ氏に連絡すると、
よしなにしてくれると思われます。

ひとまずは、皆さんのコメント待ちですかね。

didy-kpn commented on numerai-signals/new-users.md:
```diff
@@ -1,6 +1,28 @@
+現在、さまざまなレベルに対応する新しいユーザー動画やチュートリアルを作成中です。<br>
+チュートリアルが完成するまでは、ユーザーが作成したNumerai Signalsの記事を参照してください。<br>
+
+*[YouTube](https://www.youtube.com/channel/UCQt3RVSKsDpFgYIm1A-nWbA)でNumeraiを購読して、ビデオチュートリアルの通知を受け取れます。<br>
```
@titbtcqash 

*(アスタリスク) の後にスペースが入ってないため、リストになってないみたいです。
( もし不要ならすみません... )

titbtcqash commented:
@ageonsen さん　そうですね。Signalsなどの日本語訳は使うユーザーのコンセンサスが欲しいところですし、そこは多数決でいきたいです。大筋はいいとしても、しばらくは待ちですね。
@didy-kpn さん　反映しました。ついでに、大文字になっていなかったところも直しました。ありがとうございます！

didy-kpn commented on numerai-tournament/new-users.md:
```diff
@@ -1,43 +1,83 @@
+[@katsu1110さん](https://twitter.com/kk1110tt):[KagglerへのNumeraiのススメ](https://zenn.dev/katsu1110/articles/bb2b5cba9b04c9e30bfe)<br>
+不明:[【ファイナンス機械学習】著者によるNumerai解説スライドを日本語でまとめてみる](https://quantcollege.net/financial-machine-learning-numerai
+)<br>
+[@tit_BTCQASH](https://twitter.com/tit_BTCQASH):[[NumeraiのHPの見方＋サブミット＋便利リンク集](https://qiita.com/tit_BTCQASH/items/366a2d1c273507dd4b8c)<br>
```
@titbtcqash @ageonsen 

敬称はありかなしで統一してもいいと思います。

titbtcqash replied:
「さん」を消しました。ありがとうございます！

didy-kpn commented on numerai-tournament/new-users.md:
```diff
@@ -1,43 +1,83 @@
+[@blog_UKIさん](https://twitter.com/blog_uki):[機械学習による株価予測　さがそうNumerai Signals](https://qiita.com/blog_UKI/items/adba89950c4f70c2167d)<br>
+[@blog_UKIさん](https://twitter.com/blog_uki):[機械学習による株価予測　さがそう 真のNumerai Signals](https://qiita.com/blog_UKI/items/6f044b41819f1f003426)<br>
+[@katsu1110さん](https://twitter.com/kk1110tt):[KagglerへのNumeraiのススメ](https://zenn.dev/katsu1110/articles/bb2b5cba9b04c9e30bfe)<br>
+不明:[【ファイナンス機械学習】著者によるNumerai解説スライドを日本語でまとめてみる](https://quantcollege.net/financial-machine-learning-numerai
```
@titbtcqash @ageonsen 

この人は...違うのでしょうか？

https://twitter.com/developer_quant

titbtcqash replied:
仰る通りのようですね。自分では発見できていませんでした。ありがとうございます！

didy-kpn commented on numerai-tournament/new-users.md:
```diff
@@ -1,43 +1,83 @@
+* データ分析、ヒント、チュートリアルに関する投稿は、[Numeraiフォーラム](https://forum.numer.ai/)をチェックしてください。
+
+
+【Numeraiグッズのデザイン(?)】<br>
```
@titbtcqash @ageonsen 

すみません、今更かもしれませんが....このリンクは「有用な日本語記事」でも「新規参加者向けのtip」でもないと思います

titbtcqash replied:
削除しました！

didy-kpn commented on numerai-tournament/staking-and-payouts.md:
```diff
@@ -1,71 +1,69 @@
+## ペイアウト
+どれだけのNMRを得られるかはNMRのステーク量と、Corr,MMCの値に依存します。<br>
+スコアが高いほど、より多くのNMRを得ることができます。もし負のCorr,MMCとなった場合、ステークしたNMRの一部が没収され、バーンされます。<br>
+バーンとはERC-20トークンの持つ機能の一つであり、トークンを永遠に使用できなくする操作のことです。<br>
```
@titbtcqash @ageonsen 

ここは「バーンとは、トークンを永遠に使用できなくする操作のことです」ではダメでしょうか？
日本語docではNMRがERC20トークンである等の説明がないため、唐突感が否めません...。
( もし見落としていたり英語版がそうなっていた場合はすみません.... )

didy-kpn replied:
細かくて申し訳ないです！
トークンという単語もここしかないですね.... この上の行は以下の通りになってるため、「バーンとはERC-20トークンの持つ機能の一つであり、トークンを永遠に使用できなくする操作のことです」はなくてもいいと個人的には思います。

> スコアが高いほど、より多くのNMRを得ることができます。もし負のCorr,MMCとなった場合、ステークしたNMRの一部が没収され、バーンされます。<br>

もし追加されるなら、軽くでいいのでNMRはERC20トークンである旨の説明はあった方がいいのかな....

titbtcqash replied:
日本語Docにも英語Docにもバーンの説明自体がないような気がします。（あったらごめんなさい）

（Solidity使ってETHのコントラクトを出すような人が少なそうな）Numerai参加者がバーンという語を理解できると思っていないので、このように書きましたが、ご指摘の文の方がわかりやすいのでそのように修正しました！

ageonsen replied:
補足ですが、日本語版にはburn の説明というかトークンの説明自体ないです。英語版にもなかったと思います。

titbtcqash replied:
追記見ました。　　確かに、トークンという語もないですね・・。　トークンをNMRと言い換えますか？

ご提案に加え、「バーンとは、NMRを永遠に使用できなくする操作のことです。」　のみにしておけば、NMRがERC20トークンである説明もいらない感じでどうでしょう？

titbtcqash replied:
英語版がそうなのでしかたがないですが、誰もが仮想通貨に詳しい前提となっているDocsなので、強い違和感を感じています。

didy-kpn replied:
> 日本語Docにも英語Docにもバーンの説明自体がないような気がします。（あったらごめんなさい）

以下の説明は @titbtcqash さんが追加したくれたものですね。たしかにこれを除いたら説明らしい説明は見当たらないです。( 私も見落としていた場合はすみません )

> スコアが高いほど、より多くのNMRを得ることができます。もし負のCorr,MMCとなった場合、ステークしたNMRの一部が没収され、バーンされます。

didy-kpn replied:
> ご提案に加え、「バーンとは、NMRを永遠に使用できなくする操作のことです。」　のみにしておけば、NMRがERC20トークンである説明もいらない感じでどうでしょう？

はい、私はそれでいいと思います！

cshanes commented:
thank you! merging to master now
